2-ui/5-loading/02-script-async-defer/article.md

```diff
@@ -68,7 +68,7 @@ The following example demonstrates that:
 ```
 
 1. The page content shows up immediately.
-2. `DOMContentLoaded` waits for the deferred script. It only triggers when the script `(2)` is downloaded is executed.
+2. `DOMContentLoaded` waits for the deferred script. It only triggers when the script `(2)` is downloaded and is executed.
```

Comment:
I would keep the new "and" but also take out "is": "is downloaded and executed".